#ifndef COXGRAPH_SERVER_CLIENT_HANDLER_H_
#define COXGRAPH_SERVER_CLIENT_HANDLER_H_

namespace coxgraph {
namespace server {

class ClientHandler {
 public:
  ClientHandler();
  ~ClientHandler();

 private:
};

}  // namespace server
}  // namespace coxgraph

#endif  // COXGRAPH_SERVER_CLIENT_HANDLER_H_
